homeassistant/components/esphome/strings.json

```diff
@@ -49,7 +49,7 @@
   "entity": {
     "binary_sensor": {
       "call_active": {
-        "name": "Call Active"
+        "name": "[%key:component::voip::entity::binary_sensor::call_active::name%]"
```

Comment:
Should esphome be referencing voip? Would the source string not be better placed in the assist_pipeline integration?

Reply:
I guess that's better.

Reply:
Although I guess "Call Active" is maybe a bit wrong, we could call it "Assist Active" ?

Reply:
VoIP it's actually a call.